crds/AzureIngressProhibitedTarget-v1-CRD-v1.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
it seems that our e2e infra doesn't clean up some of the resources, e.g. the prohibited crd object, we might need another PR  to fix the issue if any,